File: TrackMyAct.Server/Models/Entities/InterestEntity.cs
namespace TrackMyAct.Server.Models.Entities;

public class InterestEntity : Entity
{
    public string Name { get; set; }
    public string Description { get; set; }
    public ICollection<ParticipantEntity> Participants { get; set; }
    public ICollection<EventEntity> Events { get; set; }
}

File: TrackMyAct.Mobile/composeApp/src/commonMain/kotlin/org/js/tma/dto/InterestEntity.kt
package org.js.tma.dto

import kotlinx.serialization.Serializable

@Serializable
data class InterestEntity(
    override var id: Int,
    var name: String,
    var description: String,
    var participants: List<ParticipantEntity> = emptyList(),
    var events: List<EventEntity> = emptyList()
) : Entity()